locations/jclouds/src/test/java/brooklyn/entity/rebind/persister/jclouds/BlobStoreExpiryTest.java

```diff
@@ -91,7 +91,7 @@
     private String provider;
     private String endpoint;
 
-    public synchronized BlobStoreContext getBlobStoreContext(boolean applyFix) {
+    public synchronized BlobStoreContext getBlobStoreContext() {
```

Comment:
This test assumes the endpoint will be Swift, but nothing in the class explicitly says that the test endpoint must be swift (there is just an "e.g." in some example brooklyn.properties). Would be worth being explicit about that requirement.

Note this was a problem with the code before @neykov improved it, so I'm not too bothered if it's included in this PR or another.